docs/agent/agent.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[endpoint-agent]]
+= Endpoint agent
+
+Through the Elastic Endpoint agent, Endpoint instantly detects and stops privilege escalation, defense evasion,
```

Comment:
change to "the endpoint instantly..." I will double check on case but I think that's correct.

Reply:
Actually, I believe it should be "Through the Elastic Agent, the Elastic Endpoint instantly..." we will confirm when we tag Braden/Dan on review.